src/test/java/org/damap/base/conversion/AbstractTemplateExportScienceEuropeComponentsTest.java

```diff
@@ -48,6 +56,19 @@ void determineDatasetIDsTest() {
     Assertions.assertEquals(datasetTableIDs.size(), datasets.size(), dmpDO.getDatasets().size());
   }
 
+  @Test
+  void testAddLabelForDatasetSize() {
+    DatasetDO mockDataset = new DatasetDO();
+    mockDataset.setSize(500_000_000_000L);
+    List<DatasetDO> newDatasets = new ArrayList<>();
+    newDatasets.add(mockDataset);
+    for (int i = 0; i < newDatasets.size(); i++) {
+      docVar.add(DatasetSizeRange.getLabelForSize(newDatasets.get(i).getSize()));
+    }
+    Assertions.assertEquals(1, docVar.size());
+    Assertions.assertEquals("500 - 1000 GB", docVar.get(0));
+  }
+
```

Comment:
Could you reduce this test to the essentials?
```java
Assertions.assertEquals("500 - 1000 GB", DatasetSizeRange.getLabelForSize(500_000_000_000L));
```
The other code is not really needed and it is more difficult to understand what is being tested.